ts/parts-more/GaugeSeries.ts

```diff
@@ -0,0 +1,682 @@
+     *
+     * @sample {highcharts} highcharts/plotoptions/gauge-overshoot/
+     *         Allow 5 degrees overshoot
+     *
+     * @type      {number}
+     * @default   0
+     * @since     3.0.10
+     * @product   highcharts
+     * @apioption plotOptions.gauge.overshoot
+     */
+
+    /**
+     * Options for the pivot or the center point of the gauge.
+     *
+     * In styled mode, the pivot is styled with the
+     * `.highcharts-gauge-series .highcharts-pivot` rule.
+     *
+     * @sample {highcharts} highcharts/css/gauge/
+     *         Styled mode
+     *
+     * @type    {Highcharts.CSSObject}
+     * @since   2.3.0
+     * @product highcharts
+     */
+    pivot: {},
+
+    /**
+     * The pixel radius of the pivot.
+     *
+     * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+     *         Pivot options demonstrated
+     *
+     * @type      {number}
+     * @default   5
+     * @since     2.3.0
+     * @product   highcharts
+     * @apioption plotOptions.gauge.pivot.radius
+     */
+
+    /**
+     * The border or stroke width of the pivot.
+     *
+     * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+     *         Pivot options demonstrated
+     *
+     * @type      {number}
+     * @default   0
+     * @since     2.3.0
+     * @product   highcharts
+     * @apioption plotOptions.gauge.pivot.borderWidth
+     */
+
+    /**
+     * The border or stroke color of the pivot. In able to change this,
+     * the borderWidth must also be set to something other than the default
+     * 0.
+     *
+     * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+     *         Pivot options demonstrated
+     *
+     * @type      {Highcharts.ColorString}
+     * @default   #cccccc
+     * @since     2.3.0
+     * @product   highcharts
+     * @apioption plotOptions.gauge.pivot.borderColor
+     */
+
+    /**
+     * The background color or fill of the pivot.
+     *
+     * @sample {highcharts} highcharts/plotoptions/gauge-pivot/
+     *         Pivot options demonstrated
+     *
+     * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+     * @default   #000000
+     * @since     2.3.0
+     * @product   highcharts
+     * @apioption plotOptions.gauge.pivot.backgroundColor
+     */
+
+    tooltip: {
+        headerFormat: ''
+    },
+
+    /**
+     * Whether to display this particular series or series type in the
+     * legend. Defaults to false for gauge series.
+     *
+     * @since   2.3.0
+     * @product highcharts
+     */
+    showInLegend: false
+
+// Prototype members
+}, {
+    // chart.angular will be set to true when a gauge series is present,
+    // and this will be used on the axes
+    angular: true,
+    directTouch: true, // #5063
+    drawGraph: noop,
+    fixedBox: true,
+    forceDL: true,
+    noSharedTooltip: true,
+    trackerGroups: ['group', 'dataLabelsGroup'],
+
+    /* eslint-disable valid-jsdoc */
```

Comment:
`/* eslint-disable-next-line valid jsdoc */` may be a safer alternative for this.

Reply:
It is needed for nearly every function doclet, because the `this` parameter has to be defined for TypeScript but not for our documentation. Midterm we will switch from series factory to regular class and then it is not necessary anymore to turn it off.